.gitpod.dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gitpod/workspace-full
```

Comment:
I think gitpod is really useful. Any concerns using my personal LAMMPS docker image?

Reply:
gitpod looks cool indeed, totally up to that!
On your LAMMPS docker image, yes, it was blindly copied to the release. I don't have any concerns, and I remember it working well with PySIMM.
If you are fine with that, let us leave it there for this release

Reply:
Sounds good to me!